sanitizer/_text.js

```diff
@@ -1,29 +1,20 @@
-var check = require('check-types'),
-  text_analyzer = require('pelias-text-analyzer');
+const check = require('check-types');
+const _ = require('lodash');
 
 // validate texts, convert types and apply defaults
 function sanitize( raw, clean ){
-
   // error & warning messages
-  var messages = { errors: [], warnings: [] };
+  const messages = { errors: [], warnings: [] };
```

Comment:
I don't think I'm understanding `const`, surely if something is variable such as this it's not constant?